apps/src/components/sidebar-inner-s2d.tsx

```diff
@@ -132,47 +163,67 @@ export const SidebarInnerS2D = () => {
 			<SidebarSeparator />
 
 			<SidebarMenuItem>
-				<Dropdown
+				<Dropdown<string>
 					key={fieldFrequencies.key}
 					placeholder={SelectAnOptionLabel}
 					options={fieldFrequencies.options}
 					label={fieldFrequencies.label}
-					value={fieldFrequencies.options[0].value}
+					value={frequency}
 					tooltip={tooltipFrequencies}
-					onChange={() => void 0}
+					onChange={setFrequency}
 				/>
 			</SidebarMenuItem>
 
 			<SidebarMenuItem className="mt-4">
-				<TimePeriodsControlSingle
-					tooltip={tooltipTimePeriods}
-				 />
+				<TimePeriodsControlS2D tooltip={tooltipTimePeriods} />
 			</SidebarMenuItem>
 		</>
 	);
 };
 
 SidebarInnerS2D.displayName = 'SidebarInnerS2D'
 
+/**
+ * Component displaying the release date for the sidebar.
+ *
+ * Shows a loading message if the release date is not yet available.
+ *
+ * @param locale - Locale to use for formatting the date.
+ * @param releaseDate - Release date to display.
+ */
 export const SidebarFooterReleaseDate = ({
-	date = '2025-10-06',
+	date,
 	tooltip,
+	locale = 'en',
 }: ReleaseDateProps) => {
 
```

Comment:
We sure we need to pass locale? We should have it already?

```suggestion
export const SidebarFooterReleaseDate = ({
	date,
	tooltip,
}: ReleaseDateProps) => {
	const { locale } = useLocale();
```